config-templates/azure/azuredisk-csi-driver/1.30.3/metadata.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "1.30.3",
+    "revision": "1",
+    "enabled": "true",
+    "status": "supported",
+    "description": "Azure Disk CSI Driver",
+    "tags": [
+        {
+            "name": "provider",
+            "value": "Azure"
+        },
+        {
+            "name": "offering-name",
+            "value": "azuredisk-csi-driver"
+        },
+        {
+            "name": "offering-type",
+            "value": "non-sds"
+        },
+        {
+            "name": "provider-url",
+            "value": "https://portal.azure.com"
+        },
+        {
+            "name": "support-msg",
+            "value": "For support contact Microsoft Azure"
+        },
+        {
+            "name": "provider-logo",
+            "value": "logo/azuredisk-csi-driver.svg"
+        },
+        {
+            "name": "supported-envs",
+            "value": "Azure"
+        },
+        {
+          "name": "supported-ocp-versions",
+          "value": "4.8, 4.9, 4.10, 4.11, 4.12, 4.13, 4.14, 4.15"
+        },
+        {
+            "name": "volume-type",
+            "value": "block"
+        },
+        {
+            "name": "volume-encryption-at-rest",
+            "value": "true"
+        },
+        {
+            "name": "volume-encryption-in-transit",
+            "value": "true"
+        },
+        {
+            "name": "volume-snapshot",
```

Comment:
is it not supported in this new version as well ?

Reply:
It is supported in this version and was also supported on the public version of our previous version (v1.23.0). I've not included it as I was not sure on how to test it.

Reply:
I found the relevant volume snapshot documentation and have added the corresponding resources here: https://github.com/IBM/ibm-satellite-storage/pull/441/commits/5336935a4ac800dd366a8afaf73ed20dd7d5a1d9